hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/DecommissionOMSubcommand.java

```diff
@@ -115,6 +116,8 @@ public Void call() throws IOException {
           .setHostAddress(hostInetAddress.getHostAddress())
           .build();
       omAdminProtocolClient.decommission(decommNodeDetails);
+      OMStorage omStorage = new OMStorage(ozoneConf);
+      omStorage.unsetOmNodeId();
```

Comment:
This might actually not remove/edit the entry from the version file of the decommissioned node rather will edit it on the node on which this command will be run, If the command is run on a non-OM node, it might even generate an  IOException as it wont find the version file.